app/scripts/lib/transaction/metrics.ts

```diff
@@ -978,6 +979,13 @@ async function buildEventFragmentProperties({
     uiCustomizations.push(MetaMetricsEventUiCustomization.GasEstimationFailed);
   }
 
+  if (
+    REDESIGN_TRANSACTION_TYPES.includes(transactionMeta.type as TransactionType)
+  ) {
+    uiCustomizations.push(
+      MetaMetricsEventUiCustomization.RedesignedConfirmation,
+    );
+  }
```

Comment:
I think it should also include check to see if user has enabled re-designs

Reply:
added in 50e55166aa